### ISSUE TITLE
It does not work with multiple inheritance (mixins)

### ISSUE THREAD

**Issue by mitar** [meowklaski/custom_inherit#20]
I have set of mixins to be used together with a base class:

```python
from custom_inherit import DocInheritMeta

class Parent(metaclass=DocInheritMeta(style="numpy")):
    """
    Parent class.

    Attributes
    ------------
    foo : str
        Foo attribute.
    """

class Mixin:
    """
    This is mixin which does something.
    """

class Child(Mixin, Parent):
    """
    Child description.
    """
```

```
>>> Child.__doc__
'Child description.'
```

Instead of expected:
```
>>> Child.__doc__
'Child description.\n\nAttributes\n----------\nfoo : str\n    Foo attribute.'
```

Now, the workaround is to put `Mixin` after `Parent` in the inheritance, but this goes against the logic I need (mixin has to override some things from parent).

**Comment by meowklaski** [comment]
I will fix this ASAP - I will have the metaclass descend the inheritance hierarchy rather than ascend. This will give preference to the base class over the mixins.

**Comment by mitar** [comment]
I think it is already descending in the right order, just that it finds a mixin first which does not have any numpy docstring section. What I think plugin should do is merge all sections from all ancestors. Or at least go and continue search until it finds one with at least one section.

**Comment by meowklaski** [comment]
I believe that it is ascending, which is why it finds the mixin first.
```
Child => Mixin => Parent
```

To descend would entail
```
Parent => Mixin => Child
```
Unless I am mistaken!

I'll look into merging across all ancestors.

**Comment by mitar** [comment]
OK, we use these words differently. :-) But in your terminology, I think ascending is a good direction.

**Comment by meowklaski** [comment]
I think we're on the same page. I'll try to get to this tonight.

**Comment by meowklaski** [comment]
This should be fixed with #21 . The latest version, 2.2.0, is on pypi. Please let me know if there are any issues.

**Comment by mitar** [comment]
Perfect. It works. Thanks so much.
